core/common/src/main/java/alluxio/Constants.java

```diff
@@ -205,6 +205,7 @@
   public static final String MASTER_JOURNAL_LOG_SIZE_BYTES_MAX =
       "alluxio.master.journal.log.size.bytes.max";
 
+  public static final String MASTER_FILE_ASYNC_HANDLER = "alluxio.master.file.async.handler";
```

Comment:
should "persist" be in included in the name? An "async handler" could refer to many things.